File: teste-ateliware/app/Models/Repository.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Repository extends Model
{

    //-------------------------------------------------------------------
    //          Attributes
    //-------------------------------------------------------------------

    protected $table = 'repositories';

    protected $dates = ['created_at', 'updated_at'];

    protected $fillable = [
        'full_name',
        'name',
        'owner_name',
        'description',
        'avatar_url',
        'language',
        'stars',
        'query'
    ];

    //-------------------------------------------------------------------
    //          Accessor Methods
    //-------------------------------------------------------------------
}

File: models/repository.py
from app import db

class Repository( db.Model ):
    __tablename__ = "Repositories"
    lang = db.Column( db.String, primary_key=True )
    name = db.Column( db.String )
    description = db.Column( db.String )
    link = db.Column( db.String )
    stars = db.Column( db.Integer )
    
    def __init__( self, lang, name, description, link, stars ):
        self.lang = lang
        self.name = name
        self.description = description
        self.link = link
        self.stars = stars